MBBSEmu/HostProcess/Enums/UserStatus.cs

```diff
@@ -0,0 +1,37 @@
+namespace MBBSEmu.HostProcess.Enums
+{
+    /// <summary>
+    /// User Status values, mostly taken from BRKTHU.H
+    /// </summary>
+    public enum UserStatus
```

Comment:
I also don't _THINK_ it matters, but Enum's are typed as `int` by default, but we can specify ` : ushort` just to keep the type consistent.